src/components/nav/desktop/NavHeader.jsx

```
@@ -37,13 +38,13 @@ function NavHeader({ shrink }) {
             )}
 
             <div className={`info mt-3 text-center`}>
-                <h5 className={`name`}>
+                <h1 className={`name h5`}>
                     <ImageView src={logoUrl}
                                alt={`logo`}
                                className={`img-view-logo me-1`}/>
 
                     <span dangerouslySetInnerHTML={{__html:stylizedName}}/>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)